modules/administration/pages/master-fingerprint.adoc

```diff
@@ -14,19 +24,29 @@ salt-key -F master
 ----
 +
 
-On your client, open the [path]``/etc/salt/minion`` configuration file.
+On your client, open the [path]``/etc/salt/minion.d/custom.conf`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
 Uncomment the following line and enter the master's fingerprint replacing the example fingerprint:
```

Comment:
This we should also change, as the file is created by the user then it is empty, so the user cannot "uncomment the following line". We should ask them to add the file